nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -228,20 +228,21 @@ def main() -> None:
         warnings.warn("NIXOS_INSTALL_GRUB env var deprecated, use NIXOS_INSTALL_BOOTLOADER", DeprecationWarning)
         os.environ["NIXOS_INSTALL_BOOTLOADER"] = "1"
 
+    # flags to pass to bootctl install/update
```

Comment:
This comment feels redundant, maybe? The name is kind of obvious...

Reply:
You know I actually went back and forth for a while on the comment and variable name here. My defense is that the flags aren't passed to the bootctl version and status checks.

Reply:
Ooh, good point. Wonder if there's a better name for it... `bootctl_install_flags` could work, but that would also require a comment explaining that it's passed to both. But I also think `bootctl_flags` is a tiny bit disingenuous? I don't know, maybe I'm nitpicking.

Reply:
And `flags` was kind of ambiguous now that the scope has changed and `install_and_update_flags` is annoyingly long and...

I am pretty sure there's no obvious win here and we should keep it how it is :)